### ISSUE TITLE
Tiny things

### ISSUE THREAD

**Issue by sdailey01** [StreakThunderstorm/Sonic-Unreal-Worlds#7]
1. When you're on a tiny slope you start to slide down even though you're standing still

2. Sonic can't go up easy slopes (Look at the vid to see what I mean)

3. Sonic is a little slow. (I know that can be edited easily in the settings, but I'd like to mention it.)

https://streamable.com/sxuli

edit: I also noticed that when you press the jump button in mid-air, when you already jumped, the spin ball animation resets back to the beginning position.

**Comment by StreakThunderstorm** [comment]
1. Thats the slide along surface function. It allows thew collision capsule to stop Sonic inside of blocking geometry without causing him to stop completely. I'll have to turn the function on only when Sonic's movement vector is not equal to 0.

2. I've been tweaking that but it ruins rolling. I'll have to have a separate angle range for rolling.

3. Sonic is as fast as he is in Sonic 3.

Pressing jump in mid air starts the dropdash. Hold jump and you'll charge a dropdash.

**Comment by sdailey01** [comment]
Yeah, I found out about the drop dash while messing around. Maybe add another model like how Sonic's jump turns solid to indicate that you'll roll if you hit the ground.

**Comment by sdailey01** [comment]
I found other bugs that are in this video: https://youtu.be/dR2vMMz5Bjc
